README.md

```diff
@@ -24,10 +24,31 @@ using Statistics
 lazfn = joinpath(dirname(pathof(LazIO)), "..", "test/libLAS_1.2.laz")
 pointcloud = LazIO.open(lazfn)
 
+#LAS file support is provided through LazIO.open() as well
+pointcloud = LazIO.open(somefile.las)
+```
+
+```julia
 # Index pointcloud
-cellsizes = (1.,1.)
+cellsizes = (1.,1.) #can also use [1.,1.]
 raster_index = index(pointcloud, cellsizes)
 
+#get some information about the index
+
+#the dataset the index was calculated from
+raster_index.ds
+
+#::GeoArray of point density per cell
+raster_index.counts
+
+#find highest recorded point density
+maximum(raster_index.counts)
+
+#one dimensional vector of index values joining points to cells
+raster_index.index
```

Comment:
Hmm these are kind of internal fields. Though we should probably leave this in, since it will allow users to understand what these indices are.

Reply:
It took me a second to figure out what it was while working with the package, so I thought it was worth a note at least. I don't know if point by point operations are within the scope of this particular package, so I can remove if you don't think it adds any value. I also couldn't come up with any small examples on how to use them.

Is it worth pointing out that it is a `LinearIndices` under the hood for users who may be expecting a cartesian x,y  bin syntax?

Reply:
Yeah indeed, that's why I think it's good that you added it here.

I don't think I would say that point-by-point operations are out of scope. The whole reason I started using Julia for point clouds is that it's no problem to loop over them and apply functions per point :)

The linear index would be good to mention, that may indeed throw some people off.